cmd/serve/restic/inproc.go

```diff
@@ -0,0 +1,21 @@
+package restic
+
+import (
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/rclone/rclone/cmd"
+	"github.com/rclone/rclone/cmd/serve/httplib/httpflags"
+)
+
+// NewServer returns a rclone server object which talks the restic http protocol
+func NewServer(args []string) (http.RoundTripper, error) {
+	f := cmd.NewFsSrc(args)
+	return newServer(f, &httpflags.Opt), nil
```

Comment:
I think since `cmd.NewFsSrc` is public I'd probably just make `newServer` public and then you have all the bits you need.

Reply:
Yes making newServer public would be helpful. But you may have to make server exported to keep golint quiet.